client/azure_client_set.go

```diff
@@ -168,6 +157,12 @@ func newInterfacesClient(config *clientConfig) *network.InterfacesClient {
 	return &c
 }
 
+func newStorageAccountsClient(config *clientConfig) *storage.AccountsClient {
```

Comment:
The diff looks wild but I just moved methods around to sort them alphabetically.